bug-bounty-list/bug-bounty-list.json

```diff
@@ -7599,6 +7599,16 @@
     "hall_of_fame": false,
     "safe_harbor": ""
   },
+  {
```

Comment:
This branch may need to be made up-to-date with `master`, as this went in with https://github.com/disclose/disclose/pull/24